src/app/pages/wallets/my-wallets/pages-wallets-my-wallets.component.ts

```diff
@@ -26,15 +30,65 @@ export class PagesWalletsMyWalletsComponent implements OnInit {
           data,
           isLoading: false,
           hasError: false,
+          nowEditWallet: 0,
         };
       },
       error: () => {
         this.myWalletsData = {
           data: null,
           isLoading: false,
           hasError: true,
+          nowEditWallet: 0,
         };
       },
     });
   }
+
+  public addNewWallet(userInputData_: string) {
+    this.myWalletsService.addNewWallet(userInputData_).subscribe( (wallet_: MyWallet) => {
+        this.myWalletsData.data?.push(wallet_);
```

Comment:
It would be better that way: `this.myWalletsData.data = [wallet, ...this.myWalletsData.data!]`